docs/01-programs/02-candy-machine/03-managing-candy-machines.md

```diff
@@ -79,14 +154,42 @@ Finally, it stores all items inserted in the Candy Machine and whether or not th
 - The number of items that have been loaded.
 - A list of all items that have been or will be inserted. When an item is not inserted yet, the name and URI of the item at that position are empty.
 - A bitmap — a list of yes or no — that keeps track of which items have been loaded. When this bitmap is full of yeses, all items have been loaded.
-- A list of all items that have *not* been minted yet when minting using a random order. This allows the program to grab an index at random without having to worry about picking an index that has already been minted and start again.
+- A list of all items that have _not_ been minted yet when minting using a random order. This allows the program to grab an index at random without having to worry about picking an index that has already been minted and start again.
```

Comment:
We should explain that now there is a `account_version` and `token_standard` fields on the account, so when you fetch a candy machine account you can check those values.

Reply:
I've added the `account_version` but not the `token_standard` or `ruleset` because they are documented on the previous "Candy Machine Settings" page and there's a link to reference all settings in order to avoid having to document each attribute twice.

> First of all, it stores all the settings provided when the account was created and keeps track of any changes. See the [previous page](/programs/candy-machine/candy-machine-settings) for more details on these settings.